tools/osbuilder/rootfs-builder/cbl-mariner/config.sh

```diff
@@ -6,6 +6,8 @@ OS_NAME=cbl-mariner
 OS_VERSION=${OS_VERSION:-3.0}
 LIBC="gnu"
 PACKAGES="kata-packages-uvm"
+
+[ "$UVM_BUILD_TYPE" = debug ] && PACKAGES+=" kata-packages-uvm-debug"
```

Comment:
So the idea is that there is only one kata-packages-uvm-debug package that is being used for both ConfPods and PodSandbox? Just leaving a comment here to discuss explicitly.